adapter/internal/oasparser/model/policy_container.go

```diff
@@ -182,70 +182,38 @@ func (spec *PolicySpecification) validatePolicy(policy Policy, flow PolicyFlow,
 	if !arrayContains(spec.Data.SupportedGateways, policyCCGateway) {
 		return errors.New("choreo connect gateway not supported")
 	}
-	if !spec.Data.MultipleAllowed {
+	if !spec.Data.MultipleAllowed { // TODO (renuka): remove this multiple allowed validation and compute stats
 		// TODO (renuka): check the behaviour with APIM
 		// in here allow first instance of policy to be applied if multiple is found
 		pStat := stats[policy.PolicyName]
 		if pStat.count > 1 {
 			if index != pStat.firstIndex {
 				return errors.New("multiple policies not allowed")
 			}
-			loggers.LoggerOasparser.Warnf("Operation policy \"%v\" not allowed in multiple times, appling the first policy", policy.PolicyName)
+			loggers.LoggerOasparser.Warnf("Operation policy \"%s\" not allowed in multiple times, appling the first policy", policy.PolicyName)
 		}
 	}
 
 	policyPrams, ok := policy.Parameters.(map[string]interface{})
 	if ok {
 		for _, attrib := range spec.Data.PolicyAttributes {
-			val, found := policyPrams[attrib.Name]
```

Comment:
I see that we have removed attrib validation for policies. Don't we need to?

Reply:
it is in the line 200 of newly added changes

Reply:
Will that be enough? Are you sure about removing what you already had used?

Reply:
seems like we have removed attrib key validation, that's what I meant